src/Device/Driver/OpenVario.cpp

```diff
@@ -131,13 +228,33 @@ OpenVarioDevice::POV(NMEAInputLine &line, NMEAInfo &info)
    * R: total pressure in hPa
    * S: true airspeed in km/h
    * T: temperature in deg C
+   * ?: respond with selected strings, e.g. Ballast, Bugs, Polar
+   *    Example: $POV,?,RPO,MC,WL*2E means: Real Polar, MacCready, Wing Load
    */
 
+
   while (!line.IsEmpty()) {
     char type = line.ReadOneChar();
     if (type == '\0')
       break;
 
+    if (type == '?') {
+      NullOperationEnvironment env;
+      char query_item[5];
+
+      for (int i=0;i < 10;i++) { // not more than 10 loops!
+	line.Read(query_item,sizeof(query_item));
```

Comment:
Indentation for `for` loop is missing

Reply:
This is a formatting issue on this page. Maybe with the diff.
Look at the real file:
https://github.com/bomilkar/XCSoar/blob/restucture-and-improve-openvario-driver/src/Device/Driver/OpenVario.cpp
The indentation is OK. I think.

Reply:
Oh, I guess that's because spaces and tabs are mixed for indentation. According to [xcsoar dev manual](http://download.xcsoar.org/releases/6.8.11/XCSoar-developer-manual.pdf), we should not use tabs for indentation:

> 79 columns, reasonable exceptions allowed. Indent 2 spaces, no tabs.